### PR TITLE
Add more cached items

### PR DESCRIPTION
**Pull request by Ebag333**
Add fighters, structures, and structure modules

### CODE REVIEW

**Comment by Ebag333** [comment]
This adds (or rather, keeps) the data we need for supporting fighters, citadels, and citadel modules.

This addresses the specific error listed in #10

File: eos/data/cache_generator/cleaner.py [hunk 1]
```diff
@@ -63,7 +63,10 @@ def _pump_evetypes(self):
             Category.skill,
             Category.drone,
             Category.implant,
-            Category.subsystem
+            Category.subsystem,
+            Category.fighter,
+            Category.structure,
+            Category.structureModules
```
Comment (Regner) [inline eos/data/cache_generator/cleaner.py]: Minor style nit-pick but can you please add a comma at the end of this line. It helps when doing diffs as you wont need to add it just to add another line.

For example if someone had put one at the end of the subsystems line originally then it wouldn't show up as a change now.

**Comment by Ebag333** [comment]
Closing as this is being done in #17 since the tests are throwing fits.

**Comment by Ebag333** [comment]
Reopening this PR.

Since we're going a different direction, PR #17 probably won't go in as it is, so we might as well start including this data, because we will need it sooner or later anyway.

This PR simply prevents the cache from purging fighters, structures, and structure modules, it doesn't change any functionality, and the cache size only grows by a few KB.

**Pull request closed without merging**